packages/sessions-sdk-react/src/session-provider.tsx

```diff
@@ -71,6 +70,7 @@ const ONE_DAY_IN_MS = 24 * ONE_HOUR_IN_MS;
 const DEFAULT_SESSION_DURATION = 14 * ONE_DAY_IN_MS;
 
 const ERROR_CODE_SESSION_EXPIRED = 4_000_000_000;
+const ERROR_CODE_SESSION_LIMITS_EXCEEDED = 4_000_000_008;
```

Comment:
where do you get these numbers from? why are non-consecutive?

Reply:
this error code comes from https://github.com/fogo-foundation/fogo-sessions/pull/145/files#diff-b8c5c4505cf04822f63a57243414d1ac40d4aaad09773d593ba632f026c458d9R37 which I already had to merge so I could release an upgraded sdk in order to use that error in the token program.  It's non-consecutive because there are other enum members for `SessionError` that we don't handle in the frontend, e.g. if there's an `InvalidAccountVersion` we don't do anything to handle it in the sdk.